shared/Docs/Navigation.tsx

```diff
@@ -20,14 +21,44 @@ function useInitialValue(value, condition = true) {
   return condition ? initialValue : value;
 }
 
-function TopLevelNavItem({ href, children }) {
+function TopLevelNavItem({ href, matcher, title, icon: Icon }) {
+  const router = useRouter();
+  const pathname = router.pathname;
+  const isActive = matcher.test(pathname);
+  return (
+    <NavLink href={href} isTopLevel={true}>
+      <span
+        className={clsx(
+          "flex flex-row py-1 gap-4 items-center",
+          isActive && "font-bold text-indigo-600"
+        )}
+      >
+        {Icon && (
+          <Icon className="w-5 h-5 text-slate-400 group-hover:text-slate-600 dark:group-hover:text-slate-200" />
+        )}
+        {title}
+      </span>
+    </NavLink>
+  );
+}
+
+export function TabItem({ href, children, matcher }) {
+  const router = useRouter();
+  const pathname = router.pathname;
+  const isActive = matcher.test(pathname);
   return (
-    <li className="lg:hidden">
+    <li>
       <Link
         href={href}
-        className="block py-1 text-sm text-slate-600 transition hover:text-slate-900 dark:text-slate-400 dark:hover:text-white"
+        className={clsx(
+          "text-sm leading-5 transition whitespace-nowrap px-3 py-4 relative top-0.5",
+          isActive &&
+            "font-medium text-indigo-700 dark:text-white border-b dark:border-b-white border-b-indigo-700  hover:text-indigo-900",
```

Comment:
Changing to `font-medium` based on the active state causes the tabs to get wider, I think we should make these always `font-medium`

Reply:
Thank you!